### PR TITLE
chore: update submit artwork start flow

### PR DESCRIPTION
**Pull request by MounirDhahri**
This PR resolves [ONYX-960] 

### Description

This PR updates the start submission flow to match the new designs by making the following changes:
- Update the copy and the images
- Hide start submission from my collection behind a feature flag

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- update start flow in new submit artwork flow - mounir

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[ONYX-960]: https://artsyproduct.atlassian.net/browse/ONYX-960?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (update start flow in new submit artwork flow - mounir)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against bc9ce7a8b38eef3f7b4595ebac3026c450e34171
</p>

File: src/app/Scenes/SellWithArtsy/ArtworkForm/Components/SubmitArtworkBottomNavigation.tsx [hunk 1]
```diff
@@ -41,16 +43,18 @@ export const SubmitArtworkBottomNavigation: React.FC<{}> = () => {
...
-          onPress={() => {
-            navigateToNextStep()
-          }}
-          block
-          mt={2}
-          variant="outline"
-        >
-          Start from My Collection
-        </Button>
+        {!!showStartFromMyCollection && (
```
Comment (olerichter00) [inline src/app/Scenes/SellWithArtsy/ArtworkForm/Components/SubmitArtworkBottomNavigation.tsx:46]: 👍

**Review by olerichter00 — APPROVED** [review]
Looks great!

I just realized that we should check how the whole form looks on tablets before we release it. I've created a Jira ticket: https://artsyproduct.atlassian.net/browse/ONYX-977

**Comment by MounirDhahri** [comment]
Great idea! I think we should definitely do that. We might be able to try that out early and if it works out, we can potentially use it as an inspiration for Web.

**Pull request merged**